src/Utils/utils.ts

```diff
@@ -477,3 +479,10 @@ export const mergeQueryOptions = <T extends object>(
     ),
   ];
 };
+
+export const parseOccupationFromExt = (occupation: Occupation) => {
+  const occupationObject = OCCUPATION_TYPES.find(
+    (item) => item.value === occupation
+  );
+  return occupationObject?.id;
+};
```

Comment:
Move this function to the file that uses this, since this is not a utility function that is going to be used across the entire app.